CMakeLists.txt

```diff
@@ -182,6 +182,10 @@ if(BUILD_SHARED_LIBS)
 		sentry_install(FILES "$<$<OR:$<CONFIG:Debug>,$<CONFIG:RelWithDebInfo>>:$<TARGET_PDB_FILE:sentry>>"
 			DESTINATION "${CMAKE_INSTALL_BINDIR}")
 	endif()
+else()
+	if(LINUX)
+		set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

Comment:
please use `target_compile_options` here, instead of messing with global definitions